backend/src/govsg/services/govsg-callback.service.ts

```diff
@@ -136,12 +132,14 @@ const parseTemplateMessageWebhook = async (
         messageId,
       },
     })
-    // throwing error here to return 400
+    // returning early
     // this is because callbacks could hit this endpoint before the messageId is updated in GovsgOp table
     // only do this for sending as this is unlikely to happen for other statuses
     // this will trigger a retry from WhatsApp, which will hit this endpoint again after messageId is updated
     if (whatsappStatus === WhatsAppMessageStatus.sent) {
-      throw new MessageIdNotFoundWebhookError('Message ID not found')
+      logger.error({
```

Comment:
> callbacks could hit this endpoint before the messageId is updated in GovsgOp table

In this scenario, it is acceptable to throw an error because we __need__ WhatsApp to retry sending the message right? Otherwise we would incorrectly resolve this webhook request -> update `messageId` in `GovsgOp` and then get stuck because the webhook request is never retried.

Reply:
Nice catch, let me revert this.